kingfisher_scrapy/spiders/europe_dynamic_base.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+from urllib.parse import urlsplit
+
+import scrapy
+
+from kingfisher_scrapy.base_spider import CompressedFileSpider
+from kingfisher_scrapy.util import components, handle_http_error, join
+
+
+class EuropeDynamicBase(CompressedFileSpider):
+    # SimpleSpider
+    data_type = 'record_package'
+    date_format = 'year-month'
+
+    def start_requests(self):
+        yield scrapy.Request(
+            f'{self.base_url}/ocds/services/recordpackage/getrecordpackagelist',
+            meta={'file_name': 'list.json'},
+            callback=self.parse_list
+        )
+
+    @handle_http_error
+    def parse_list(self, response):
+        urls = response.json()['packagesPerMonth']
+        for url in reversed(urls):
+            path = urlsplit(url).path
+            if self.from_date and self.until_date:
+                # URL looks like https://www.zppa.org.zm/ocds/services/recordpackage/getrecordpackage/2016/7
+                year, month = map(int, url.rsplit('/', 2)[1:])
+                url_date = datetime.datetime(year, month, 1)
+                if not (self.from_date <= url_date <= self.until_date):
+                    continue
+            yield self.build_request(self.base_url+path, formatter=join(components(-2), extension='zip'))
```

Comment:
```suggestion
            yield self.build_request(f'{self.base_url}{path}', formatter=join(components(-2), extension='zip'), priority=number * -1)
```

With this, we are requesting the latest data first.